heka/sandbox/outputs/redshift.lua

```diff
@@ -76,15 +76,48 @@ local env = assert (driver.postgres())
 local con, err = env:connect(db_name, db_user, db_password, db_host, db_port)
 assert(con, err)
 assert (con:execute[[
-        CREATE TABLE IF NOT EXISTS logs(
-        timestamp BIGINT,
-        severity INTEGER,
-        pid INTEGER,
-        hostname VARCHAR(126),
-        type VARCHAR(126),
-        logger VARCHAR(126),
-        envversion VARCHAR(126),
-        payload VARCHAR(126)
+        CREATE TABLE IF NOT EXISTS telemetry_sample_42 (
+            "msg_Timestamp"               TIMESTAMP,
```

Comment:
Don't need the quotes; also all these fields are lowercased by Redshift so we may just want to make them match the end result here.